cpp/src/fil/fil.cu

```diff
@@ -915,12 +1011,30 @@ void from_treelite(const raft::handle_t& handle, forest_t* pforest,
 void from_treelite(const raft::handle_t& handle, forest_t* pforest,
                    ModelHandle model, const treelite_params_t* tl_params) {
   const tl::Model& model_ref = *(tl::Model*)model;
-  model_ref.Dispatch([&handle, pforest, tl_params](const auto& model_inner) {
+  model_ref.Dispatch([&](const auto& model_inner) {
     // model_inner is of the concrete type tl::ModelImpl<T, L>
     from_treelite(handle, pforest, model_inner, tl_params);
   });
 }
 
+template <typename T, typename L, typename N>
+char* sprintf_shape(const tl::ModelImpl<T, L>& model, storage_type_t storage,
```

Comment:
It's worth documenting that the pointer returned is allocated with `malloc()`, and that it's the responsibility of the caller to free it using `free()`.